src/lib/utils.ts

```
@@ -71,3 +71,16 @@ export function isRivenSeason(item: RivenItem): item is RivenSeason {
+
+export function getFormattedTime(time: string) {
+	const date = new Date(time);
+	return date.toLocaleDateString('en-US', {
+		year: 'numeric',
+		month: 'long',
+		day: 'numeric',
+	}) + ' ' + date.toLocaleTimeString('en-US', {
+		hour: 'numeric',
+		minute: 'numeric',
+		second: 'numeric'
+	});
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)